Dockerfile

```
@@ -0,0 +1,18 @@
+FROM openjdk:11
+WORKDIR /app
```

```suggestion
WORKDIR /app
LABEL org.opencontainers.image.source=https://github.com/gravwell/ocsf-translation-tools
```

Setting the above label will help GitHub tie the image to this repository. It makes it a little easier to hop between the GHCR package list and this repo.

I think it would also be good to add a few labels to support documentation and such. You can find documentation for different labels here: https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys

If it were me, I'd consider ...

- `org.opencontainers.image.title` - the name of the thing
- `org.opencontainers.image.documentation` - a link to the README would be great
- `org.opencontainers.image.authors` - credit where credit's due
- `org.opencontainers.image.description` - always good to have a little description of what the image is

I'd really only insist on setting `source` and `description`. The others are just suggestions. Feel free to set or omit them as you please!

---

This is very interesting, just updated.

---

Looks great!